tests/test_integration.py

```diff
@@ -1,12 +1,24 @@
 EXPECTED_OUTPUT = """
-arbitrum
+bsc
 ├── mainnet
-│   ├── alchemy
 │   └── geth  (default)
 ├── testnet
-│   ├── alchemy  (default)
-│   └── geth
+│   └── geth  (default)
+└── local  (default)
+    └── test  (default)
+ethereum  (default)
+├── mainnet
+│   └── geth  (default)
+├── ropsten
+│   └── geth  (default)
+├── kovan
+│   └── geth  (default)
+├── rinkeby
+│   └── geth  (default)
+├── goerli
+│   └── geth  (default)
 └── local  (default)
+    ├── geth
```

Comment:
We should make these tests work better in a local environment when the runner may or may not have additional plugins installed.

Similar to the work done in `tests/integratons/test_networks.py` in this PR https://github.com/ApeWorX/ape/pull/920/files

Reply:
I found that I could used the same testing logic as fantom and it didn't throw an error when I added additional plugins. https://github.com/ApeWorX/ape-fantom/blob/main/tests/test_integration.py